cellfinder_napari/train/train.py

```diff
@@ -0,0 +1,184 @@
+from lib2to3.pgen2.token import OP
+from pathlib import Path
+from typing import Optional
+
+from cellfinder_core.download.models import model_weight_urls
+from cellfinder_core.train.train_yml import models
+from magicgui import magicgui
+from magicgui.widgets import FunctionGui, PushButton
+from napari.qt.threading import thread_worker
+from napari.utils.notifications import show_info
+
+from cellfinder_napari.utils import (
+    header_label_widget,
+    html_label_widget,
+    widget_header,
+)
+
+from .train_containers import (
+    MiscTrainingInputs,
+    OptionalNetworkInputs,
+    OptionalTrainingInputs,
+    TrainingDataInputs,
+)
+
+
+def train() -> FunctionGui:
+    @magicgui(
+        header=header_label_widget,
+        training_label=html_label_widget("Network training", "h3"),
```

Comment:
I think we should use the `*` operator in the `html_label_widget` function definition such that `tag` is a keyword argument
```suggestion
        training_label=html_label_widget("Network training", tag="h3"),
```